Neos.ContentRepository/Classes/Domain/Repository/NodeDataRepository.php

```diff
@@ -1207,7 +1207,7 @@ protected function filterOutRemovedObjects(array &$objects)
      * @param array $nodes NodeData including removed entries
      * @return array Only those NodeData instances which are not flagged as removed
      */
-    protected function withoutRemovedNodes(array $nodes)
+    public function withoutRemovedNodes(array $nodes)
```

Comment:
I need access to these functions, should I make them public or do something else?

Reply:
Moved methods to utility class.